meta.yml

```
@@ -3,16 +3,19 @@ fullname: Stalmarck
 shortname: stalmarck
 organization: coq-community
 community: true
+travis: true
 
 synopsis: Proof of Stålmarck's algorithm in Coq
 
 description: |
   A two-level approach to prove tautologies using Stålmarck's algorithm in Coq.
 
+doi: 10.1007/3-540-44659-1_24
+
 paper:
-  doi: 10.1007/3-540-44659-1_24
-  url: https://www.irif.fr/~letouzey/download/stalmarck.ps.gz
-  title: Formalizing Stålmarck's Algorithm in Coq
+  pub_doi: 10.1007/3-540-44659-1_24
```

```suggestion
- pub_doi: 10.1007/3-540-44659-1_24
```
This needs to be a list item now.